src/timeline-list/index.tsx

```diff
@@ -82,6 +83,7 @@ const TimelineList = (props: TimelineListProps) => {
             {...timelineProps}
             key={item}
             date={item}
+            scrollToNow={scrollToNow && index === INITIAL_PAGE}
```

Comment:
The 'date' passed to 'timelineProps' determines the date of the middle page (the INITIAL_PAGE), this date is not necessarily today's date, so you won't be able to scroll to now (the current time on today's date). So I'm not sure your condition is correct... Maybe you need to check that this page is today's page? Did I understand correctly am I missing something?

Reply:
Yes, that's a good point. 
I checked again with the product, the behavior might be a little different. 
Ill change it and update

Reply:
Ok, I fixed the logic.
First of all, I added a new prop - `initialTime` that defines the initial time to scroll to
In TimelineList I do the following
- If initial page is "today" - I pass the scrollToNow given by the user
- If the initial page is not "today" - I pass the `initialTime` prop 

The reset of the timeline pages (any page that is not the initial) should be sync accordingly, based on the `scrollOffset` prop

Reply:
I find it a little bit confusing.. I'd guess that the initialTime will work for all cases, even for the current day

Reply:
Also, I think it's weird that the `scrollToNow` prop works only when the `initialPage` is the current day.
I think we can have only one prop - `initialTime` and the user will decide whether to pass the current time or any other time..

Reply:
TBH, these requirements came from Bookings product, but they do make sense.. 
Also, I changed the logic a little more since the last time.
There are few "initial scroll" logics, I'll explain them from the most prioritized to the least

- **scrollToNow** - will scroll to current time. Specifically, In the context of TimelineList it's relevant only when the initial page is "today"
- **scrollToFirst** - will scroll to the first event of the day (if there are events that day). Again, in the context of TimelineList it's relevant only if the initial page is NOT "today"
- **initialTIme** - Will scroll to an initial time (usually represent the business opening hours) - and like before, In the context of TimelineList, it's relevant only if the initial page is NOT "today" 

Logically it make sense to scroll to the current time only when you're in "today", otherwise the current time has no meaning. 
And again, all this logic refers to the initial page and the initial scroll. This logic won't take affect when moving between the pages/timelines in the list

Reply:
I think the 2 last types should work for **any** initial page (today or not) since they have a very specific intent that does not exclude today's page (which will be the initial page in most cases), while the 'scrollToNow' should work only if the current page is today since it doesn't make sense otherwise (as it is currently).